tests/test_fga.py

```diff
@@ -534,3 +534,14 @@ def test_query(self, mock_query_response, mock_http_client_with_response):
             warrant_token="warrant_token",
         )
         assert response.dict(exclude_none=True) == mock_query_response
+
+    def test_query_with_context(self, mock_query_response, mock_http_client_with_response):
+        mock_http_client_with_response(self.http_client, mock_query_response, 200)
+
+        response = self.fga.query(
+            q="select member of type user for permission:view-docs",
+            order="asc",
+            warrant_token="warrant_token",
+            context={"region": "us", "subscription": "pro"},
+        )
+        assert response.dict(exclude_none=True) == mock_query_response
```

Comment:
Can we make an assertion on what the request URL looks like? (i.e. it should include a string-ified version of the JSON context)